awx_collection/tests/integration/targets/tower_workflow_template/tasks/main.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: Create an SCM Credential
+  tower_credential:
+    name: SCM Credential for JT
+    organization: Default
+    kind: scm
+
+- name: Create a Demo Project
+  tower_project:
+    name: Job Template Test Project
+    organization: Default
+    state: present
+    scm_type: git
+    scm_url: https://github.com/ansible/ansible-tower-samples.git
```

Comment:
Weird to have this git project different from the others. I will reiterate my idea that, alternatively, we could use `https://github.com/ansible/awx.git` to host playbooks for these tests. :trollface: